### PR TITLE
Update get_image_info function

### PR DESCRIPTION
**Pull request by hsmj1412**
Try to use the better way to get the info

Signed-off-by: Junxiang Li <junli@redhat.com>

### CODE REVIEW

File: virttest/utils_misc.py [hunk 1]
```diff
@@ -3102,9 +3103,8 @@ def get_image_info(image_file):
     """
     try:
         cmd = "qemu-img info %s" % image_file
-        if compare_qemu_version(2, 10, 0):
-            # Currently the qemu lock is only introduced in qemu-kvm-rhev,
-            # if it's introduced in qemu-kvm, will need to update it here.
+        if libvirt_storage.check_qemu_image_lock_support():
```
Comment (xutian) [inline virttest/utils_misc.py]: image lock is not libvirt special feature, condication here is not suitable.

Comment (chunfuwen) [inline virttest/utils_misc.py]: Agree with xutian, so it need self-define one function to query whether lock feature is support or not

Comment (hsmj1412) [reply virttest/utils_misc.py]: @xutian Where do you think it should be?utils_misc?

Comment (hsmj1412) [reply virttest/utils_misc.py]: Done

**Comment by chunfuwen** [comment]
@hsmj1412 ,after this PR is merged, probably create new PR to remove check_qemu_image_lock_support method from previous libvirt_storage.py

File: virttest/utils_misc.py [hunk 2]
```diff
@@ -3100,11 +3100,19 @@ def get_image_info(image_file):
                            'dsize' : '931135488',
                            'csize' : '65536'}
     """
+    def check_qemu_image_lock_support():
+        cmd = "qemu-img"
+        try:
+            binary_path = process.system_output("which %s" % cmd)
```
Comment (lmr) [inline virttest/utils_misc.py]: You should use avocado.utils.path.find_command, which does the same and does not require calling a subprocess.

Comment (lmr) [reply virttest/utils_misc.py]: Or virttest.utils_misc.find_command, which amounts to the same.

**Comment by hsmj1412** [comment]
@chunfuwen @lmr Updated

File: virttest/utils_misc.py [hunk 3]
```diff
@@ -3100,11 +3100,16 @@ def get_image_info(image_file):
                            'dsize' : '931135488',
                            'csize' : '65536'}
     """
+    def check_qemu_image_lock_support():
```
Comment (chunfuwen) [inline virttest/utils_misc.py]: Move this implementation to individual method, rather than wrapper in this method

Comment (hsmj1412) [reply virttest/utils_misc.py]: @chunfuwen Updated

**Pull request merged**